src/main/java/com/aws/greengrass/logging/impl/config/LogConfig.java

```diff
@@ -23,6 +23,9 @@
 
     private static final LogConfig INSTANCE = new LogConfig();
 
+    /**
+     * This is the root logger configuration. Child loggers may refer to this to find default config values
+     */
     public static LogConfig getInstance() {
         return INSTANCE;
```

Comment:
I might suggest just rename this directly `ROOT_LOG_CONFIG` and `getRootLogConfig`

Reply:
Yes that's clearer.